### ISSUE TITLE
Staging Review finding: Follow one thing per page pattern

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#127623]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** BIO - Pingwind
**Product:** VA Form 21-8940 Veteran's Application for Increased Compensation Based on Unemployability
**Feature:** Form Digitization

## Findings details
**VA.gov Experience Standard - issue:** [User experience is not consistent with the VA.gov design principles.](https://design.va.gov/about/experience-standards/#users-experience-is-consistent-with-the-vagov-design-principles)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @allison0034 (Design), @jasonday (Accessibility)

### Description

Throughout the form, there are pages that contain many form fields asking multiple types of questions and sometimes contain complex sets of questions that are conditionally revealed. This can present a cognitive burden for users.

## Recommended action

Walk through the form and ensure each screen is covering one concept only. [One thing per page](https://design.va.gov/patterns/ask-users-for/a-single-response#what-is-one-thing-per-page) doesn't necessarily mean "one input element per page" but it does mean breaking up the conceptual topics covered on each page.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** [Ask user for single response](https://design.va.gov/patterns/ask-users-for/a-single-response)
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** User flow
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
